LocalReferenceLinkRel/explainer.md

```diff
@@ -0,0 +1,105 @@
+# Local References In <link> Tags
+
+## Authors:
+
+- Andy Luhrs
+- Kurt Catti-Schmidt
+
+## Participate
+- [Issue tracker](https://github.com/MicrosoftEdge/MSEdgeExplainers/labels/LRLR)
+- [Discussion forum](https://github.com/whatwg/html/issues/11019)
+
+## Status of this Document
+
+This document is intended as a starting point for engaging the community and
+standards bodies in developing collaborative solutions fit for standardization.
+As the solutions to problems described in this document progress along the
+standards-track, we will retain this document as an archive and use this section
+to keep the community up-to-date with the most current standards venue and
+content location of future work and discussions.
+
+* This document status: **Active**
+* Expected venue: [WHAT Working Group](https://whatwg.org/)
+* Current version: this document
+
+## Introduction
+Modern web development practices have converged towards building reusable components instead of building monolithic documents. Frameworks such as React and Angular have popularized this approach, which has made its way natively to the web platform via Web Components. 
+
+Web Components allow for defining custom elements, and these custom elements often rely on Shadow DOM for ID encapsulation. But Shadow DOM doesn't just encapsulate ID's - it encapsulates style as well. The current preferred approach
+for styling elements under a Shadow DOM is through `<link>` tags, but these tags have several downsides, namely that they must be an external file or a dataURI. The performance and developer ergonomics of this
```

Comment:
```suggestion
for styling elements under a Shadow DOM is through `<link>` tags, but these tags have several downsides, namely that they must be an external file or a dataURI. The performance of an external file and developer ergonomics of generating a dataURI make them difficult to use. Because of that, developers using Custom Elements have expressed strong interest in addressing these issues.
```

Nit: Make it explicit why it's not ideal.